### PR TITLE
Bug fixes in DB and server calls working.

### PR DESCRIPTION
**Pull request by Ashcoca**
@simonyuanliu @CZosa @mlymolina Fixed some bugs and got my server stuff working, please review whoever is interested. I think we need to have another code review done by Wednesday.

### CODE REVIEW

File: client/src/components/carousel.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+/*
+Ways to implement this module:
+1.  Conditional rendering. Load multiple sets of restaurant cards and
+    render them as you click back/forward
+2.  handleClickBack/Next will load the next 3 cards or previous 3 
+3.
+*/
+
+class Carousel extends React.Component {
+  // constructor() {
```
Comment (simonyuanliu) [inline client/src/components/carousel.jsx]: constructor and super should have argument props buddy. please add them, if you are thinking about using a state to hold your pictures.

File: client/src/index.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import $ from 'jquery';
+
+
+class App extends React.Component {
```
Comment (simonyuanliu) [inline client/src/index.jsx]: I know you are only on the planning stage. Remember to use stateless component if you only render and return stateless element.    😃

File: server/server.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+const express = require('express');
+const port = 3000;
```
Comment (simonyuanliu) [inline server/server.js]: Please save port 3000 for proxy server. Can you choose another port for your app?

File: client/src/index.jsx (hunk 2 above)
Comment (Ashcoca) [reply client/src/index.jsx]: Thanks, I need to remember to use those more often!

File: server/server.js (hunk 3 above)
Comment (Ashcoca) [reply server/server.js]: Ooh, that's right. I always just set 3000 by default, I'll switch to another.

**Pull request merged**